includes/general.php

```diff
@@ -1052,7 +1052,15 @@ function pods_shortcode_run( $tags, $content = null ) {
 
 		return $pod->form( $tags['fields'], $tags['label'], $tags['thank_you'] );
 	} elseif ( ! empty( $tags['field'] ) ) {
-		if ( empty( $tags['helper'] ) ) {
+		if ( ! empty( $tags['template'] ) ) {
+			// @todo Single Pods query based on relationship field.
+			$return  = '';
+			$related = $pod->field( $tags['field'], array( 'output' => 'pods' ) );
```

Comment:
This would only work on a related object that is also a pod itself.

But if that was the case, you'd probably want to use `'output' => 'find'` instead and just do:

```php
if ( $related instanceof Pods ) {
	$return = $related->template( $tags['template'], $content );
}
```

Reply:
I want to note that calling it like this won't need a foreach or while, since template does that when it detects it's in a list context (without passing a direct ID to the Pods object).

Reply:
True, but in this case (since we use `field`) we get a list of separate Pods objects.

Reply:
If we use `'output' => 'find'` it would be more performant. Multiple Pods objects ends up being more intensive.